sssom/cliques.py

```diff
@@ -18,41 +18,42 @@ def to_networkx(msdf: MappingSetDataFrame) -> nx.DiGraph:
     # m = {
     #    "owl:subClassOf",
     # }
-    for mapping in doc.mapping_set.mappings:
-        s = mapping.subject_id
-        o = mapping.object_id
-        p = mapping.predicate_id
-        # TODO: this is copypastad from export_ptable
-
-        pi = None
-
-        if p == "owl:equivalentClass":
-            pi = 2
-        elif p == "skos:exactMatch":
-            pi = 2
-        elif p == "skos:closeMatch":
-            # TODO: consider distributing
-            pi = 2
-        elif p == "owl:subClassOf":
-            pi = 0
-        elif p == "skos:broadMatch":
-            pi = 0
-        elif p == "inverseOf(owl:subClassOf)":
-            pi = 1
-        elif p == "skos:narrowMatch":
-            pi = 1
-        elif p == "owl:differentFrom":
-            pi = 3
-        elif p == "dbpedia-owl:different":
-            pi = 3
-
-        if pi == 0:
-            g.add_edge(o, s)
-        elif pi == 1:
-            g.add_edge(s, o)
-        elif pi == 2:
-            g.add_edge(s, o)
-            g.add_edge(o, s)
+    if doc.mapping_set.mappings is not None:
+        for mapping in doc.mapping_set.mappings:
+            s = mapping.subject_id
+            o = mapping.object_id
+            p = mapping.predicate_id
+            # TODO: this is copypastad from export_ptable
```

Comment:
TODO indeed. please refactor the code from export_ptable and reuse it

Reply:
@matentzn ?

Reply:
This is TODO for another PR, sorry for the confusion